src/element.rs

```diff
@@ -114,14 +114,41 @@ impl IconElement {
         let width = icon_type.pixel_width();
         let height = icon_type.pixel_width();
         match icon_type.encoding() {
-            #[cfg(feature = "pngio")]
```

Comment:
This line needs to be restored so that the crate can still build with the `pngio` feature disabled.